cmd/load-generator/latency-charter.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/python
+
+import matplotlib
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+import numpy as np
+import datetime
+import json
+import pandas
+import matplotlib
+import argparse
+import os
+matplotlib.style.use('ggplot')
+
+# sacrifical plot for single legend
+matplotlib.rcParams['figure.figsize'] = 1, 1
+randFig = plt.figure()
+randAx = plt.subplot()
+randAx.plot(0, 0, color='green', label='good', marker='+')
+randAx.plot(0, 0, color='red', label='failed', marker='x')
+randAx.plot(0, 0, color='black', label='sent', linestyle='--')
+randAx.plot(0, 0, color='green', label='50th quantile')
+randAx.plot(0, 0, color='orange', label='90th quantile')
+randAx.plot(0, 0, color='red', label='99th quantile')
+handles, labels = randAx.get_legend_handles_labels()
+
+# big ol' plotting method
```

Comment:
Probably because this is WIP and I'm not especially familiar with matplotlib but I found this function a little bit hard to follow. As a utility for generating graphs it probably doesn't need tremendous polish but some comments along the way describing what you're doing with matplotlib would go a long way.